syntax = "proto3";

package Commands;

option java_package = "com.interfaces.proto";
option java_outer_classname = "RegisterUserCommandProto";

message RegisterUserCommand
{
    int32 user_id = 1;
    int32 vehicle_id = 2;
    string vehicle_key = 3;
}